recipes/underworld/meta.yaml

```
@@ -0,0 +1,84 @@
+{% set name = "underworld" %}
+{% set version = "2.14.1" %}
+{% set sha256 = "ec922eb6d10d6a6031d5c76b106df0c79b122ee61c43900866f2017bf819b2eb" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/ad/44/e491e7faa66cd927b722304d3e0a7ecc5f6c24cefa6044868a84548303d5/underworld-{{ version }}b0.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [win or osx]
+  number: 0
+  script: PETSC_DIR=$PREFIX {{ PYTHON }} -m pip -vv install --no-deps .  
+  # shared objects to ignore when checking linkage information
+  missing_dso_whitelist:
+    - "**/site-packages/underworld/lib/**"
+
+
+requirements:
+
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ mpi }}
```

does MPI need to be in the build section too? can you try without it? You likely need 
```suggestion
    - {{ mpi }}  # [build_platform != target_platform and mpi != 'nompi']
```

as is done in the HDF5 feedstock https://github.com/conda-forge/hdf5-feedstock/blob/main/recipe/meta.yaml#L78

---

Yes MPI is needed for the build. So I'll leave it in for now.